src/__test__/faqspage.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import { render, cleanup, fireEvent } from "@testing-library/react";
+import Enzume, {mount} from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16';
+import Faq from '../pages/FAQpage/Faqs'
+
+
+Enzume.configure({adapter: new Adapter() });
+
+afterEach(cleanup);
+
+describe("describe if faq page work as expected", () => {
+  const toggleFaq = jest.fn
+  const faq= {
+    question: 'How accurate is the min calcuration ?',
+    ansuer: 'None don\'t address hardware issue',
+    open: false
+  }
+  let wrapper;
+  beforeEach(()=>{
+    wrapper = mount(<Faq faq={faq} toggleFaq={toggleFaq} />);
+  })
+  it("render ", () => {
+    expect(wrapper).not.toBeNull(); 
+  });
+  it("switch Answers ", () => {
+    const { getByTestId } = render(<Faq faq={faq} toggleFaq={toggleFaq} />);
+    fireEvent.click(getByTestId("faqs"));
+  });
+ 
+
+
+})
```

Comment:
The changes look good but there are still some newline eslint errors. Like here and in `src/shared/styles/FaqsStyle.js`
I suggest using prettier vs code extension as it automatically adds the newlines

Reply:
oky let me do it

Reply:
i made the  changes

Reply:
@IRANKUND the coverage dropped to much. You did not test your code. You increase the coverage to at least 92%